### ISSUE TITLE
Can't add tools to favorites(add a context menu when long press app tile)

### ISSUE THREAD

**Issue by ghost** [laiiihz/alga#96]
**Describe the bug**
If i hold on any tool tile it should show me an option to save that tool or tools to favorites, but its not showing anything.

**To Reproduce**
1. Hold onto any tool's tile

**Expected behavior**
It should show me an option to add that tool to favorites.

**Screenshots**
![Screenshot_20231124-131002_Alga](https://github.com/laiiihz/alga/assets/103308339/9018ea75-e1a9-46f4-8e28-9c36c9ff94df)

**Additional context**
If i hold and select too many tiles(or one), i have to manually deselect all tiles by again holding onto them. Please make it like if i press back button, it deselect all the selected tiles.

**Comment by laiiihz** [comment]
yeah, this makes sense, i'll add a context menu when long press the tile.
